packages/gatsby-theme-newrelic/src/components/Tabs/BarItem.js

```diff
@@ -39,17 +39,16 @@ const BarItem = ({
           color: var(--primary-text-color);
         }
 
-        ${isSelected &&
-        css`
+        &.isSelected {
           color: var(--primary-text-color);
           border-bottom: var(--brand-button-primary-accent) solid 3px;
 
           .dark-mode & {
             border-bottom: var(--brand-button-primary-accent-hover) solid 3px;
           }
-        `}
+        }
       `}
-      className={className}
+      className={`${className || ''}${isSelected ? ' isSelected' : ''}`}
```

Comment:
what do you think about using [`classnames`](https://www.npmjs.com/package/classnames) for this? it's [v small](https://bundlephobia.com/package/classnames@2.3.2)

Reply:
I was thinking about classnames while i did this! 🙈 v small is true, let's bring it in